src/richie/apps/courses/lms/__init__.py

```diff
@@ -25,9 +25,7 @@ def select_lms(url):
             return None
 
         for lms_configuration in settings.LMS_BACKENDS:
-            if re.match(
-                lms_configuration.get("SELECTOR_REGEX", r".*"), url
-            ) and re.match(lms_configuration.get("COURSE_REGEX", r".*"), url):
+            if re.match(lms_configuration.get("COURSE_REGEX", r".*"), url):
```

Comment:
You are removing the ability to detect several instances of a LMS? Is it useful to support that?

Reply:
This idea here is that the same regex can serve to select a LMS and to extract the course ID so why not simplify? I don't see what possibilities this change has removed?

Reply:
As long as no server-to-server queries are made between richie and the LMS there is no impact.

Reply:
As discussed IRL, it is in fact possible to use the same regex for LMS selection AND course id extraction.